doc/rtd/howto/pgp.rst

```diff
@@ -0,0 +1,201 @@
+.. _pgp:
```

Comment:
If running this guide, don't be like me and forget to install this branch into the container and then be mad it doesn't work at the end.